pkg/templates/templates.go

```diff
@@ -42,13 +48,13 @@ func SetAPIResources(apiresList []*metav1.APIResourceList) {
 	kubeAPIResourceList = apiresList
 }
 
-// HasTemplate performs a simple check for the template delimiter of "{{" to
```

Comment:
there should be a way for  the processes to be able to set the delimiters as initial\global  configuration just like the kube-client.

So it isnt set everytime and on everyone of the calls that need it.

There needs to be two sets of methods , one takes the delims and uses that and another that doesn't take the value and uses the global if available and fails if not .

go probably has a way to specify a parameter optionally